src/haddock/modules/topology/topoaa/defaults.yaml

```
@@ -95,6 +95,26 @@ tolerance:
   long: Percentage of allowed failures for a module to successfully complete
   group: module
   explevel: expert
+cyclicpept_dist:
+  default: 2.0
+  type: integer
+  min: 0
+  max: 9999999999999999
+  title: Cutoff distance for cyclisation
+  short: Cutoff distance in Angstroms for identification of backbone cyclic peptide
+  long: Cutoff distance in Angstroms for identification of backbone cyclic peptide
+  group: molecule
+  explevel: guru
+disulphide_dist:
+  default: 2.0
+  type: integer
+  min: 0
+  max: 9999999999999999
```

In this case, I would increase the default to `2.20`, as the distance observed in `1SFI` is 2.05.
I think it is nice if automatically detected.
```suggestion
  default: 2.20
  type: float
  precision: 2
  min: 0.00
  max: 999999.99
```